cg/services/order_validation_service/workflows/fastq/models/order.py

```diff
@@ -0,0 +1,8 @@
+from cg.constants import DataDelivery
+from cg.services.order_validation_service.models.order_with_samples import OrderWithNonHumanSamples
+from cg.services.order_validation_service.workflows.fastq.models.sample import FastqSample
+
+
+class FastqOrder(OrderWithNonHumanSamples):
```

Comment:
The OrderWithNonHumanSamples is a typing alias - not a parent class. So I do not think that this inheritance will work. I would also like all order types to only inherit from the parent class Order, so as to make the code as readable as possible and avoid the OrderInSample stacking of inheritances.

Reply:
The `OrderWithNonHumanSamples` is defined as a class in `cg/services/order_validation_service/models/order_with_samples.py` and it is used as a parent class for the Microbial and Mutant orders already

Reply:
Sorry, this was a very confused comment added yesterday where I got the Sample alias confused - you are completely correct